Doc/library/sqlite3.rst

```diff
@@ -1591,6 +1574,22 @@ Cursor objects
       including :abbr:`CTE (Common Table Expression)` queries.
       It is only updated by the :meth:`execute` and :meth:`executemany` methods.
 
+   .. attribute:: row_factory
+
+      Control how a row fetched from this :class:`!Cursor` is represented.
+      Set to :class:`sqlite3.Row` or a custom :term:`callable`
+      that accept two arguments,
+      a :class:`Cursor` object and the raw row results as a :class:`tuple`,
+      and return a custom object representing an SQLite row.
```

Comment:
```suggestion
      Set to :class:`sqlite3.Row`;
      or a custom :term:`callable` that accepts two arguments,
      a :class:`Cursor` object and a :class:`tuple` of the raw row results,
      and returns a custom object representing an SQLite row.
```

I suggest tweaking the wording and syntax here to make clearer that the details on the arguments and return value is only directly applicable to the custom callable (since the user isn't passing or accepting the arguments to `sqlite.Row`, and it is returning a `Row` instance, rather than a generic "custom" object, which in the same sentence is explicitly contrasted with the `Row` class).

Also, I suggested rewording the `tuple` bit, since the previous wording was a bit ambiguous and required multiple passes to parse whether elements of the `tuple` were the `Cursor` object and a row results, or just the individual row results.

Reply:
SGTM